integration-test/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -19,9 +19,11 @@
 import org.ehcache.CacheManager;
 import org.junit.Test;
 
+// CSOFF: AvoidStaticImport
```

Comment:
Given how this works, I'd rather have the sample in its own class file named to indicate this is a static import example.
Otherwise we risk other samples to start using static imports silently.